distribution/docs/src/main/resources/content/_managing/_configuring/web-context-policy-manager.adoc

```diff
@@ -23,7 +23,7 @@ The guest principal will be used either by itself or along with any other princi
 
 === Session Storage
 Enabling session storage allows Jetty to enable the JSESSIONID cookie to maintain the user login.
```

Comment:
✏️ This part isn't necessarily true anymore right. JSESSIONID stuff is always enabled we just don't look at it anymore